mnefun/_mnefun.py

```diff
@@ -1417,18 +1418,17 @@ def save_epochs(p, subjects, in_names, in_numbers, analyses, out_names,
             # now make evoked for each out type
             evokeds = list()
             for name in names:
-                if p.weighting:
-                    #TODO give access to _chpi_snr_epochs args?
+                if p.average_weighting == 'chpi':
                     weights = _chpi_snr_epochs(e[name])
```

Comment:
@kambysese The important thing is that the cHPI signals are unaltered in the data that is used for computing the weights. I'm not yet 100% sure what's happening here, but if the data is movement compensated, the cHPI signals may indeed be distorted (or totally gone).